lib/cnet/cnet/cnet.c

```diff
@@ -77,8 +78,14 @@ cnet_config_create(uint32_t num_chnls, uint32_t num_routes)
         if (cnet_route4_create(cnet, num_routes, 0) < 0)
             CNE_ERR_GOTO(leave, "Failed to create route\n");
 
+        if (cnet_route6_create(cnet, num_routes, 0) < 0)
```

Comment:
do we need to check if (CNET_ENABLE_IP6) before we call `cnet_route6_create()`

Reply:
Moved the check into the cnet_route6_create() function.